### ISSUE TITLE
Rules: links with role none, role slider still flagging improper use of link

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#1748]
This code:
```
<a role="none"><span class="nav-drop-title-wrap">eBranch<span class="dropdown-nav-toggle"><span class="kadence-svg-iconset svg-baseline"><svg aria-hidden="true" class="kadence-svg-icon kadence-arrow-down-svg" fill="currentColor" version="1.1" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><title>Expand</title><path d="M5.293 9.707l6 6c0.391 0.391 1.024 0.391 1.414 0l6-6c0.391-0.391 0.391-1.024 0-1.414s-1.024-0.391-1.414 0l-5.293 5.293-5.293-5.293c-0.391-0.391-1.024-0.391-1.414 0s-0.391 1.024 0 1.414z"></path> </svg></span></span></span></a>
```

Is still flagging Improper Use of link on Mountain Home (in the nav). Version 1.43.0 of Accessibility Checker.

Also on Highland (https://highland.edu/finearts/prairie-wind/), these links have a slider role:
```
<a class="mejs-horizontal-volume-slider edac-highlight-element-selected" href="javascript:void(0);" aria-label="Volume Slider" aria-valuemin="0" aria-valuemax="100" aria-valuenow="100" role="slider"><span class="mejs-offscreen">Use Up/Down Arrow keys to increase or decrease volume.</span><div class="mejs-horizontal-volume-total"><div class="mejs-horizontal-volume-current" style="left: 0px; width: 100%;"></div><div class="mejs-horizontal-volume-handle" style="left: 100%;"></div></div></a>
```

This is not an ideal implementation (an input would be better), but it is technically correct and common in older media players.

I would have thought that https://github.com/equalizedigital/accessibility-checker/pull/1698 might have fixed this, but maybe not.

**Comment by mjoslyn** [comment]
@pattonwebz - I can pick this up
